src/spec/index.spec.ts

```diff
@@ -1143,10 +1143,10 @@ describe('lilconfig', () => {
             );
             const relativeFilepath = filepath.slice(process.cwd().length + 1);
 
-            expect(lilconfig('test-app').load(relativeFilepath)).rejects.toThrowError(
+            await expect(lilconfig('test-app').load(relativeFilepath)).rejects.toThrowError(
                 'Unexpected token # in JSON at position 2',
             );
-            expect(cosmiconfig('test-app').load(relativeFilepath)).rejects.toThrowError(
+            await expect(cosmiconfig('test-app').load(relativeFilepath)).rejects.toThrowError(
```

Comment:
to avoid

```
ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From src/spec/index.spec.ts.

      at loader (node_modules/cosmiconfig/src/loaders.ts:37:12)
      at Explorer.loadFileContent (node_modules/cosmiconfig/src/Explorer.ts:81:32)
      at Explorer.createCosmiconfigResult (node_modules/cosmiconfig/src/Explorer.ts:89:36)
      at runLoad (node_modules/cosmiconfig/src/Explorer.ts:104:33)
      at cacheWrapper (node_modules/cosmiconfig/src/cacheWrapper.ts:13:18)
```